debian/control

```diff
@@ -0,0 +1,15 @@
+Source: clearwater-sip-stress
```

Comment:
No, it's the name of the source package (https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-Source) - we only produce binary packages so this doesn't matter. See https://github.com/Metaswitch/clearwater-net-snmp/blob/master/debian/control#L1 for something that does produce a source package and doesn't match the repo.